app/controllers/api/v1/recipes_controller.rb

```diff
@@ -13,4 +13,18 @@ def index
     end
       render json: @recipes, :include => [:user]
   end
+
+  def create
+    @recipe = Recipe.new(recipe_params)
+    @recipe.save
+    @recipe
+
+  end
+
+  def recipe_params
+   params.require(:recipe).permit(:title, :snippet, :difficulty, :duration, :servings, :user_id )
```

Comment:
If we allow `user_id` as a param here, a malicious user could create recipes under any user's ID. Under the circumstances, that's probably not a big deal, but best security practices should prevent us from allowing it.